nautobot_floor_plan/app-config-schema.json

```diff
@@ -11,7 +11,26 @@
         "default_y_axis_labels": {
             "type": "string",
             "default": "numbers"
-        }
-    },
+        },
+        "default_statuses": {
+            "type": "object",
+            "properties": {
+                "FloorPlanTile": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "color": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                }
+            }
+        },
     "additionalProperties": false
-}
+    }
```

Comment:
Don't let the spacing fool you here, this nests `additionalProperties` inside of `properties`. It was originally at the top level (same as `$schema`, `$id`, etc).

```json
{
    "$schema": "https://json-schema.org/draft/2020-12/schema",
    "$id": "https://raw.githubusercontent.com/nautobot/nautobot-app-floor-plan/develop/nautobot_floor_plan/app-config-schema.json",
    "$comment": "TBD: Update $id, replace `develop` with the future release tag",
    "type": "object",
    "properties": {
        "default_x_axis_labels": {
            "type": "string",
            "default": "numbers"
        },
        "default_y_axis_labels": {
            "type": "string",
            "default": "numbers"
        },
        "default_statuses": {
            "type": "object",
            "properties": {
                "FloorPlanTile": {
                    "type": "array",
                    "items": {
                        "type": "object",
                        "properties": {
                            "name": {
                                "type": "string"
                            },
                            "color": {
                                "type": "string"
                            }
                        }
                    }
                }
            }
        }
    },
    "additionalProperties": false
}
```